packages/owl-bot/test/database.ts

```diff
@@ -42,29 +47,30 @@ describe('database', () => {
     }
   });
 
-  it('store configs', async () => {
+  it('stores and retrieves configs', async () => {
     const db = admin.firestore();
     const store = new FirestoreConfigsStore(db, 'test-');
-    const repo = 'googleapis/' + uuidv4();
+    const repoA = 'googleapis/' + uuidv4();
+    const repoB = 'googleapis/' + uuidv4();
     const dockerImageA = uuidv4();
     const dockerImageB = uuidv4();
 
     // Confirm that the new repo and dockerImage aren't stored yet.
-    const noConfigs = await store.getConfigs(repo);
+    const noConfigs = await store.getConfigs(repoA);
```

Comment:
overtime it would be nice to pull together a mock config store for a set of smoke tests.

Reply:
Agreed.